dashboard/src/components/OperatorInstanceForm/OperatorInstanceForm.v2.tsx

```diff
@@ -40,7 +40,7 @@ export function parseCSV(
   crdName: string,
   setIcon: (icon: string) => void,
   setCRD: (crd: IClusterServiceVersionCRD) => void,
-  setDefaultValues: (v: string) => void,
+  setDefaultValues?: (v: string) => void,
```

Comment:
What's the reason for marking optional here? (similar to yesterdays PR?)

Reply:
because I am reusing the function in the `OperatorInstance` component and there I am not interested in the `defaultValues` of the custom resource (so I don't have a function to `setDefaultValues`).